MEME/MEME/Artist/ArtistHomeViewController.swift

```diff
@@ -8,10 +8,101 @@
 import UIKit
 
 class ArtistHomeViewController: UIViewController {
-
+    @IBOutlet var artistHomeProfileStatusView: UIView!
+    @IBOutlet var artistHomeProfileNoStatusView: UIView!
+    @IBOutlet var artistProfileImageView: UIImageView!
+    @IBOutlet var artistHomeProfileLabel: UILabel!
+    
+    @IBOutlet var artistReservationStatusTableView: UITableView!
+    
+    // 더미 데이터
+    private var makeUpNameArray : [String] = ["메이크업1","메이크업2","메이크업3"]
+    private var modelNameArray : [String] = ["모델1","모델2","모델3"]
+    private var reservationDateArray : [String] = ["2024.01.01 월","2024.01.02 화","2024.01.03 수"]
+    
+    
+    // 예약 전적 유무
+    private var isReservation : Bool = true
+    // 오늘 예약 유무
+    private var isTodayReservation : Bool = false
+    // 내일 예약 수
+    private var tomorrowRes : Int = 3
+    
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        
+        UISet()
+        tableViewConfigure()
+    }
+    private func UISet(){
+        
+        if isTodayReservation{
+            artistHomeProfileNoStatusView.isHidden = true
+            artistHomeProfileStatusView.isHidden = false
+        }else{
+            artistHomeProfileNoStatusView.isHidden = false
+            artistHomeProfileStatusView.isHidden = true
+        }
```

Comment:
```suggestion
        artistHomeProfileNoStatusView.isHidden = isTodayReservation
        artistHomeProfileStatusView.isHidden = !isTodayReservation
```
위와 같이 사용할 수도 있으니 참고해주세요!